assets/scss/components/_navigation.scss

```diff
@@ -242,6 +248,14 @@
     flex-direction: column;
     justify-content: center;
     align-items: center;
+
+    @include bp(medium) {
+        margin-top: rem-calc(432px);
```

Comment:
Does this value have a meaning (e.g. is it related to another element)? If it does, maybe consider moving it to a variable?